pykokkos/core/visitors/workunit_transformer.py

```diff
@@ -0,0 +1,18 @@
+from _ast import FunctionDef
+from ast import NodeTransformer
+from typing import Any
+
+class RemoveTransformer(NodeTransformer):
```

Comment:
Please add a docstring that explains the purpose of this class. Look at the docstrings for other classes that have them

Reply:
Deleted this, as this was only needed for functors